firmware/common2015/utils/logger/logger.cpp

```diff
@@ -47,6 +55,27 @@ void log(uint8_t logLevel, const char* source, int line, const char* func,
         time_t sys_time = time(NULL);
         strftime(time_buf, 25, "%H:%M:%S", localtime(&sys_time));
 
+        // if first call to log, find the right file to edit
+        if (logNameInd > 1) findSmallestLogFile();
+
+        FILE* fp = fopen(LOG_FILE_NAMES[logNameInd], "a");
+
+        if (fp) {
+            fseek(fp, 0L, SEEK_END);
+
+            if (ftell(fp) > maxLogSize) {
+                // Deletes contents of file
```

Comment:
This comment implies that you are deleting the current fp... can you make it a bit more clear that you will overwrite the next opened file?